src/new.rs

```diff
@@ -89,13 +98,24 @@ pub fn create_new_project<P: AsRef<Path>>(dest: P) -> Result<()> {
     Ok(())
 }
 
-pub fn create_new_document(doc_type: &str, name: &str, config: &Config) -> Result<()> {
+pub fn create_post(name: &str, config: &Config) -> Result<()> {
     let path = Path::new(&config.source);
-    let full_path = &path.join(&config.posts).join(name);
+
+    let local = Local::now().format("%d %b %Y %H:%M:%S %z").to_string();
+    let vector: Vec<u8> = Vec::from(new_post_tmpl);
+    let mut out = String::from_utf8(vector).unwrap();
+    out = out.replace("%title%", name);
+    out = out.replace("%date%", &local);
```

Comment:
fmt seems to support [named arguments](https://doc.rust-lang.org/std/fmt/#named-parameters).  Eventually we want to allow the skeleton files to be user-provided.